route/ton/src/types.rs

```diff
@@ -0,0 +1,89 @@
+use candid::CandidType;
+use ic_stable_structures::storable::Bound;
+use ic_stable_structures::Storable;
+use omnity_types::{TicketId, Token, TokenId};
+use serde_derive::{Deserialize, Serialize};
+use std::borrow::Cow;
+
+#[derive(CandidType, Deserialize, Serialize, PartialEq, Eq, Clone, Debug)]
+pub struct PendingTicketStatus {
+    pub ton_tx_hash: Option<String>,
+    pub ticket_id: TicketId,
+    pub seq: u64,
+    pub error: Option<String>,
+    pub pending_time: u64,
+}
+
+#[derive(CandidType, Deserialize, Serialize, PartialEq, Eq, Clone, Debug)]
+pub struct PendingDirectiveStatus {
+    pub ton_tx_hash: Option<String>,
+    pub seq: u64,
+    pub error: Option<String>,
+}
+
+impl Storable for PendingDirectiveStatus {
+    fn to_bytes(&self) -> Cow<[u8]> {
+        let mut bytes = vec![];
+        let _ = ciborium::ser::into_writer(self, &mut bytes);
```

Comment:
Consider using bincode for serialization and deserialization
```suggestion
        let bytes = bincode::serialize(&self).expect("failed to serialize");
        Cow::Owned(bytes)
```